style.css

```diff
@@ -0,0 +1,353 @@
+body {
+  padding: 0;
+  margin: 0;
+  background-color: #fff;
+  width: 100%;
+  height: 100%;
+}
+
+li {
+  list-style: none;
+  display: inline-block;
+}
+
+.flex-container {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: stretch;
+}
+
+.flex-container-2 {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  flex-grow: 1;
+  max-height: 100%;
+  height: 100%;
+  max-width: 100%;
+  width: 100%;
+}
+
+.flex-container-3 {
+  display: flex;
+  justify-content: center;
+}
+
+.flex-container-4 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.flex-container-5 {
+  display: flex;
+  flex-wrap: wrap;
+  justify-items: stretch;
+  align-content: center;
+  justify-content: center;
+  margin-top: 10px;
+}
+
```

Comment:
- [ ] DRY stands for "Don't Repeat Yourself." The main idea with DRY in CSS is to avoid repeating the same bits of code in our style sheet. You are repeating display:flex multiple times please fix it to keep our code DRY.

Reply:
Done :heavy_check_mark:

Reply:
Well resolved